mopidy/core/tracklist.py

```diff
@@ -205,9 +205,10 @@ def index(self, tl_track):
         The position of the given track in the tracklist.
 
         :param tl_track: the track to find the index of
-        :type tl_track: :class:`mopidy.models.TlTrack`
+        :type tl_track: :class:`mopidy.models.TlTrack` or :class:`None`
```

Comment:
Only because the current code allowed it, and some tests fail if this is strict.